### PR TITLE
Add Rich Glazerman to Data Core Team

### PR DESCRIPTION
**Pull request by richgt**

### CODE REVIEW

**Review by runspired — APPROVED** [review]

**Review by IgnaceMaes — COMMENTED** [review]
Could you make a square crop of your picture? It seems like the website doesn't handle non-square pictures very nicely.

You could use e.g. an online tool like https://edit.photo/.

**Comment by richgt** [comment]
> Could you make a square crop of your picture? It seems like the website doesn't handle non-square pictures very nicely.
> 
> You could use e.g. an online tool like [edit.photo](https://edit.photo/).

Done

**Review by jaredgalanis — APPROVED** [review]

File: tests/unit/controllers/teams-test.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module('Unit | Controller | teams', function (hooks) {
     // Create an intermediate data structure for assertion
     const output = dataTeamMembers.map((teamMember) => teamMember.id);
 
-    assert.deepEqual(output, ['chris-thoburn', 'krystan-huffmenne']);
+    assert.deepEqual(output, ['chris-thoburn', 'krystan-huffmenne', 'rich-glazerman']);
```
Comment (IgnaceMaes) [inline tests/unit/controllers/teams-test.js:68]:
```suggestion
    assert.deepEqual(output, [
      'chris-thoburn',
      'krystan-huffmenne',
      'rich-glazerman',
    ]);
```

**Comment by IgnaceMaes** [comment]
Thank you, and congrats! 🎉

**Pull request merged**